Source/Public/Set-VSTeamWorkItemType.ps1

```diff
@@ -0,0 +1,92 @@
+function Set-VSTeamWorkItemType {
+   [CmdletBinding(DefaultParameterSetName='LeaveAlone',SupportsShouldProcess=$true,ConfirmImpact='High')]
+   param(
+      [parameter(ValueFromPipelineByPropertyName=$true)]
+      [vsteam_lib.ProcessTemplateValidateAttribute()]
+      [ArgumentCompleter([vsteam_lib.ProcessTemplateCompleter])]
+      $ProcessTemplate = $env:TEAM_PROCESS,
```

Comment:
What makes this better than using the built-in PSDefaultParameterValues that the rest of the code uses for ProjectName?

Reply:
See long comment above :-)